src/julia_internal.h

```diff
@@ -1301,6 +1277,9 @@ void jl_push_excstack(jl_task_t *ct, jl_excstack_t **stack JL_REQUIRE_ROOTED_SLO
                       jl_value_t *exception JL_ROOTED_ARGUMENT,
                       jl_bt_element_t *bt_data, size_t bt_size);
 
+// System util to get maximum RSS
+JL_DLLEXPORT size_t jl_maxrss(void);
```

Comment:
Should this be here or in `julia.h` considering it is `JL_DLLEXPORT`ed?

Reply:
Not really? There are a bunch of DLLExported function in `julia-internal.h`, including system-ish utils such as RNG.